### DISCUSSION TITLE
Compatibility

### DISCUSSION THREAD

**Discussion by micro-fx** [Donkie/Spoolman#131] [General]
Is it possible to use spoolman, even if you dont use mainsail, klipper, etc.
and be able to manually add/edit filaments?

**Comment by Donkie** [comment]
Yes, Spoolman is completely independent.

**Reply by micro-fx** [reply]
thank you for the info @Donkie . i managed to get it running. however i am now trying to add Disane87's Homeassistant integration, and mentions a API-key from Spoolman. where or how do you get this?

**Reply by Donkie** [reply]
Spoolman has no concept of API keys so not sure what he means by that.

**Reply by micro-fx** [reply]
ok thanks for confirming, i now asked him on his github.

![image](https://github.com/Donkie/Spoolman/assets/48091146/309355f8-a8a9-41f3-b01c-46c302918b90)

**Reply by Disane87** [reply]
I will remove the API -key portion. Don't know why I implemeted this. Sorry!

**Reply by Disane87** [reply]
You can close this since the HA integration has a new version without api key :)
